stdlib/experimental/usage/usage.flux

```diff
@@ -178,6 +262,46 @@ from = (
 // usage.limits(host: "https://us-west-2-1.aws.cloud2.influxdata.com", orgID: "x000X0x0xx0X00x0", token: token)
 // ```
 //
+// ### Output organization limits in a table
```

Comment:
I had good reason to use this some weeks ago. The more you know! :rainbow::star2: